src/main/webapp/WEB-INF/views/extracost.jsp

```diff
@@ -0,0 +1,38 @@
+<%@ page session="false"%>
+<%@ taglib prefix="spring" uri="http://www.springframework.org/tags"%>
+<%@ taglib prefix="form" uri="http://www.springframework.org/tags/form"%>
+
+<html>
+<body>
+
+<form:form method="POST" action="/extracost/Add" modelAttribute="extracost">
```

Comment:
Uwaga na to `Add` po zmianie endpointa :)

Reply:
Zrobione.